tools/etcd-dump-logs/main.go

```diff
@@ -38,10 +38,14 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	defaultEntryTypes string = "Normal,ConfigChange"
+)
+
 func main() {
 	snapfile := flag.String("start-snap", "", "The base name of snapshot file to start dumping")
 	index := flag.Uint64("start-index", 0, "The index to start dumping")
-	entrytype := flag.String("entry-type", "", `If set, filters output by entry type. Must be one or more than one of:
+	entrytype := flag.String("entry-type", defaultEntryTypes, `If set, filters output by entry type. Must be one or more than one of:
```

Comment:
comment that the default type is Normal and ConfigChange.

Reply:
updated. PTAL